### ISSUE TITLE
[BUG]  claude code flickering spinner triggers epilepsy

### ISSUE THREAD

**Issue by til2000** [anthropics/claude-code#91286]
### Preflight Checklist

- [x] I have searched [existing issues](https://github.com/anthropics/claude-code/issues?q=is%3Aissue%20state%3Aopen%20label%3Abug) and this hasn't been reported yet
- [x] This is a single bug report (please file separate reports for different bugs)
- [x] I am using the latest version of Claude Code

### What's Wrong?

the flickering colorchanging behavior triggers epilepsy - there's no documented approach to stop it from moving.

### What Should Happen?

there should be config to show it in a non colorchanging, non moving manner.

### Error Messages/Logs

```shell

```

### Steps to Reproduce

.

### Is this a regression?

No, this never worked

### Claude Code Version

latest

### Platform

Other

### Operating System

Ubuntu/Debian Linux

### Terminal/Shell

Other
